tracer/src/Datadog.Tracer.Native/module_metadata.h

```diff
@@ -122,6 +123,27 @@ class ModuleMetadata
         }
         return debuggerTokens.get();
     }
+
+    void EnsureInitSanityTypeSpecToken()
```

Comment:
What does ensuring sanity mean? May need a comment to explain.

Reply:
Figured out it's not needed, deleted in https://github.com/DataDog/dd-trace-dotnet/pull/3505/commits/bd6273cab072678d95f56045183b51aa7a22aaee.

Reply:
Resolving.